### PR TITLE
removed dead instances

### PR DESCRIPTION
**Pull request by ihatespawn**
image
you still didn't do it so i did it for you

### CODE REVIEW

**Comment by hyperdefined** [comment]
I didn't do it as I didn't have time to yet,

File: backend/instances [hunk 1]
```diff
@@ -67,42 +55,28 @@ dl-api.memorybouquet.com,dl.memorybouquet.com,https
 cobapi.fly.dev,None,https
 api.sacreations.me,md.sacreations.me,https
 cobapi.imanity.de,cobalt.imanity.de,https
-cobaltapi-oracle-bbb-arm-01.thetech.network,cobalt.thetech.network,https
```
Comment (hyperdefined) [inline backend/instances:70]: This one should be left on the list. I just added it, but the owner has cloudflare protection on so it blocks my server (they have high security on). Been in contact with them, just waiting.

Comment (ihatespawn) [reply backend/instances]: done

File: backend/instances [hunk 2]
```diff
@@ -9,14 +9,11 @@ cobalt.canine.tools,cobalt.canine.tools,https
...
 api.co.rooot.gay,co.rooot.gay,https
 wukko.wolfdo.gg,co.wolfdo.gg,https
-co-api.blueb.me,co.blueb.me,https
 downloadapi.stuff.solutions,download.stuff.solutions,https
 cobalt.api.timelessnesses.me,cobalt.timelessnesses.me,https
 api-dl.cgm.rs,dl.cgm.rs,https
-cobalt.synzr.space,cobalt.synzr.space,https
 co-api.mae.wtf,co.mae.wtf,https
 capi.oak.li,c.oak.li,https
-co.tskau.team,co.tskau.team,https
```
Comment (hyperdefined) [inline backend/instances:19]: This one is back online.

File: backend/instances [hunk 3]
```diff
@@ -67,42 +55,28 @@ dl-api.memorybouquet.com,dl.memorybouquet.com,https
...
 # instances without domains
 31.220.77.64:9000,31.220.77.64:9001,http
 188.165.137.180:9000,188.165.137.180:9001,http
 168.119.212.145:1010,None,http
 2.56.214.74:9000,None,http
-77.120.21.10:3201,None,http
 92.51.47.196:9000,None,http
-95.140.158.39:9000,None,http
-161.35.207.247:9000,None,http
-86.195.21.136:8018,None,http
```
Comment (hyperdefined) [inline backend/instances:81]: This one is back online.

Comment (ihatespawn) [reply backend/instances:81]: https://github.com/ilikespawn/CobaltTester/commit/ca520f0

File: backend/instances (hunk 2 above)
Comment (ihatespawn) [reply backend/instances:19]: https://github.com/ilikespawn/CobaltTester/commit/ca520f0

**Comment by ihatespawn** [comment]
image
(i used Upside Down Mode so if they're up i get a notification and they're marked as down in uptime kuma)
i'll update this pr (of if you merge it before then i'll create a new one) if any of them are up

**Review by hyperdefined — APPROVED** [review]

**Pull request merged**